abdurahmon/db.sql

```diff
@@ -0,0 +1,171 @@
+CREATE TABLE if not exists students (
+    student_id SERIAL PRIMARY KEY,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    gpa FLOAT
```

Comment:
1. gpa kerak emas o'rniga enrollments table'dagi gradelarni hisoblab gpa chiqariladi birodar
2. enrollment_year kerak sababi tasklarda enrollment_year bilan ishlanadi
3. table to'g'irlab table'ga moslab tasklarni bajaring